Device/_Example_Flash/Abstract.txt

```diff
@@ -0,0 +1 @@
+Flash Programming Algorithm for STMicroelectronics STM32G4xx 512 KB Flash
```

Comment:
The old abstract contained much more text. Did you reduce this by intent?

Reply:
There was no 'old' abstract. This example did not exists before and the abstract describes what the example is about.

Reply:
Ah, okay.. perhaps I messed example and template.